src/raft_server.cxx

```diff
@@ -1260,7 +1263,7 @@ bool raft_server::request_leadership() {
 
 void raft_server::become_follower() {
     // stop hb for all peers
-    p_tr("  FOLLOWER\n");
+    p_in("[BECOME FOLLOWER]");
```

Comment:
It will be helpful if this log shows the term number as well:
```c++
p_in("[BECOME FOLLOWER] term %lu", state_->get_term());
```

Reply:
thx